src/screens/Text.js

```diff
@@ -193,7 +190,8 @@ export default () => (
           <PropertyValue type="string">
             <Description>
               If truncate = tip, the full text content will be placed in a Tip
-              that will appear on hover.
+              that will appear on hover. The alignment of tip can be done using
+              the `dropProps`.
```

Comment:
```suggestion
              that will appear on hover.
```
I'm thinking we should leave the description for `truncate` as is and instead add the `dropProps` to the documentation as a separate prop. In the `dropProps` description we should state that it should only be used when `truncate='tip'`.